### ISSUE TITLE
rename to datacheck_conditions

### ISSUE THREAD

**Issue by maxnoe** [fact-project/fact_conditions#1]
I think this is a better name, not?

**Comment by tarrox** [comment]
The point of the package is not just to make easy use of datacheck files, but to also store them for use for others. So the package is primary for fact use. So maybe `fact_datacheck_conditions`?

**Comment by dneise** [comment]
Should it be a module in pyfact maybe?

**Comment by maxnoe** [comment]
> So the package is primary for fact use. 

The package resides in `fact-project`

> Should it be a module in pyfact maybe?

Yes, that would have the advantage of being pip installable and not spamming pypi with more fact packages

**Comment by tarrox** [comment]
I agree that it would make sense to make it part of pyfact as it basically uses it heavily. Question is how to handle the supplied condition sets, as part of the idea behind this package was to also have a central hub for all the different Condition Sets used by us.
Basically have them in sub folders of the condition folder, like this: `conditions/thesis/bulinski/crab1314_no_scatter_light.yaml`.

So the question is, do we want this option and if yes how? Maybe just keep a seperate project?
